### DISCUSSION TITLE
注册wangeditor-next/plugin-markdown 怎么让与toolbarConfig菜单 做联动

### DISCUSSION THREAD

**Discussion by chenxq-297** [wangeditor-next/wangEditor-next#460] [General]
## 问题描述

**已经完成markdown插件注册，怎么配置菜单栏，或者说定义菜单的class里面 需要怎么开关markdown插件（以及markdown的预览模式）**

## wangEditor 版本

    "@wangeditor-next/editor": "^5.6.15",
    "@wangeditor-next/editor-for-vue": "^5.1.14",
    "@wangeditor-next/plugin-markdown": "^1.0.4",

## 查阅了文档 

注册组件：https://cycleccc.github.io/docs/guide/development#%E5%8A%AB%E6%8C%81%E7%BC%96%E8%BE%91%E5%99%A8%E4%BA%8B%E4%BB%B6%E5%92%8C%E6%93%8D%E4%BD%9C-%E6%8F%92%E4%BB%B6
自定义菜单类:https://cycleccc.github.io/docs/guide/development#selectmenu
自定义添加菜单：https://cycleccc.github.io/docs/guide/toolbar-config#insertkeys）

## 最小成本的复现步骤

1.安装以上版本依赖
2.Boot.registerPlugin(markdownModule)
3. ....

## 目前卡在的问题
![image](https://github.com/user-attachments/assets/61abc26f-a950-488e-bd42-15da5804fa93)

**Comment by cycleccc** [comment]
菜单类的实现可以参考以下代码
https://github.com/cycleccc/wangEditor-next/blob/3b4b06f82d7ec29335b305a63bce307b8970f26b/packages/plugin-link-card/src/module/menu/index.ts#L12

**Reply by chenxq-297** [reply]
我详情看了源码 但是有点令我无从下手 
具体原因是因为 markdown插件以及IDomEditor中没有提供对应开关的api
有什么思路或是方法吗

**Reply by chenxq-297** [reply]
![image](https://github.com/user-attachments/assets/f6b633bc-6a9f-47d0-a32c-73828de9dd9e)
目前的代码

**Comment by cycleccc** [comment]
Markdown 菜单实现可参考 link-card 插件中的菜单模块。本讨论现关闭。
